build/primary/main/func-time-series-bundle-ingestor.tf

```
@@ -35,6 +35,9 @@ module "time_series_bundle_ingestor" {
     WEBSITE_RUN_FROM_PACKAGE                            = 1
     WEBSITES_ENABLE_APP_SERVICE_STORAGE                 = true
     FUNCTIONS_WORKER_RUNTIME                            = "dotnet-isolated"
+    STORAGE_ACCOUNT_NAME                                = data.azurerm_key_vault_secret.st_shared_data_lake_name.value
+    STORAGE_CONNECTION_STRING                           = data.azurerm_key_vault_secret.kvs_st_data_lake_primary_connection_string.value
```

Would we want to align the naming with that of the shared resources?

```suggestion
    DATA_LAKE_ACCOUNT_NAME                                = data.azurerm_key_vault_secret.st_shared_data_lake_name.value
    DATA_LAKE_CONNECTION_STRING                           = data.azurerm_key_vault_secret.kvs_st_data_lake_primary_connection_string.value
```